examples/bzlmod/extensions.bzl

```diff
@@ -0,0 +1,39 @@
+"""
+This module is for setting the pyhton_interpreter_target argument
+when we use pip.parse in MODULE.bazel.  This extension includes a
+build rule that creates a file named python.cfg.  The rules_python
```

Comment:
Hm, writing a little config like like this seems like it might give us some other options? Because we could drop some extra info into it for the later pip repo rule to read.

But, also: why do we have to write a file? Why can't we pass in the target directly? e.g.,

```
use_repo(python, "python3_x86_linux")
pip.parse(interpreter = "@python3_x86_linux//:interpreter")
```

Reply:
you can, but then you are hardcoding it for Linux